File: Ejercicio 6/Ejercicio 6.py
NAME = "Renzo Leonardo Cienfuegos Cardenas"
GROUP = 2

# Solicitamos la cantidad de elementos del vector
n = int(input("Ingrese la cantidad de elementos del vector: "))

# Creamos el vector vacío
vector = []

# Solicitamos el primer elemento del vector
primer_elemento = float(input("Ingrese el elemento 1: "))

# Agregamos el primer elemento al vector
vector.append(primer_elemento)

# Inicializamos el mínimo con el primer elemento
minimo = primer_elemento

# Solicitamos los elementos restantes del vector
for i in range(1, n):

    elemento = float(input(f"Ingrese el elemento {i + 1}: "))

    # Agregamos el elemento al vector
    vector.append(elemento)

    # Verificamos si el elemento actual es menor que el mínimo
    if elemento < minimo:

        minimo = elemento

# Mostramos el resultado
print(f"\nEl elemento mínimo del vector es: {minimo}")

File: Ejercicio 6/Ejercicio 6.cpp
#include <iostream>
using namespace std;

int main() {

    string NAME = "Renzo Leonardo Cienfuegos Cardenas";
    int GROUP = 2;

    int n;

    // Solicitamos la cantidad de elementos del vector
    cout << "Ingrese la cantidad de elementos del vector: ";
    cin >> n;

    int vector[n];

    // Solicitamos el primer elemento del vector
    cout << "Ingrese el elemento 1: ";
    cin >> vector[0];

    // Inicializamos el mínimo con el primer elemento
    int minimo = vector[0];

    // Solicitamos los elementos restantes del vector
    for (int i = 1; i < n; i++) {

        cout << "Ingrese el elemento " << i + 1 << ": ";
        cin >> vector[i];

        // Verificamos si el elemento actual es menor que el mínimo
        if (vector[i] < minimo) {

            minimo = vector[i];

        }

    }

    // Mostramos el resultado
    cout << "\nEl elemento minimo del vector es: "
         << minimo << endl;

    return 0;
}
